lib/pinyin/parsers.ex

```diff
@@ -8,7 +8,7 @@ defmodule Pinyin.Parsers do
   # Conversions #
   # ----------- #
 
-  defp char_to_integer(c), do: [c] |> to_string() |> String.to_integer()
+  defp char_to_integer(c), do: c - ?0
```

Comment:
Curious in what sense this is better?

Reply:
It just avoids a string conversion and a parsing step by converting the codepoint straight to it's integer equivalent. It's not really related to the PR, but cleaned it up while I was at it.